api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricDataParams.java

```diff
@@ -19,15 +19,20 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import com.wordnik.swagger.annotations.ApiModel;
+import com.wordnik.swagger.annotations.ApiModelProperty;
+
 /**
  * @author John Sanda
  */
+@ApiModel
 public class MetricDataParams {
 
     protected String tenantId;
     protected String name;
     protected Map<String, String> tags = new HashMap<>();
 
+    @ApiModelProperty(required = true, value = "Tenant's unique identifier")
     public String getTenantId() {
```

Comment:
I'm not sure this field is used actually. The tenant information is read from as a path param. Did I miss something?